Face_contours/STEP/full_face/_STEP-full-face_model_notes.txt

```diff
@@ -0,0 +1,10 @@
+2020-02-06
```

Comment:
Is this date correct?

Reply:
@darraghbr nope, you're 100% correct. I mistyped